### ISSUE TITLE
请问2000年以后的数据更新了吗？

### ISSUE THREAD

**Issue by liuxue666** [imhuster/Enterprise-Registration-Data-of-Chinese-Mainland#21]

**Comment by imhuster** [comment]
yes. 为了下载方便，我已经将相应文件分别保存在  **[Enterprise-Registration-Data/csv](https://github.com/imhuster/Enterprise-Registration-Data-of-Chinese-Mainland/tree/csv)、[Enterprise-Registration-Data/xls](https://github.com/imhuster/Enterprise-Registration-Data-of-Chinese-Mainland/tree/xls)、[Enterprise-Registration-Data/json](https://github.com/imhuster/Enterprise-Registration-Data-of-Chinese-Mainland/tree/json) 三个分支**中了。
